docs/guide/custom-functions.md

```diff
@@ -78,11 +79,13 @@ Function names are case-insensitive, as they are all normalized to uppercase.
 :::
```

Comment:
The text above this line contains too many repetitions with the section "Function name translations". Can you shorten the text and put a link to the section "Function name translations" for details?

Also the section "Localizing custom functions" on the page `localizing-functions.md` repeats the same information in a very vague way. It would be good to replace the information on that page with a link to the "Function name translations" section as well.

Reply:
Good point. I reduced the section `3. Add your function's names` to the bare minimum and added a reference to the section `Function name translations`. Also `localizing-functions` guide now redirects users to `custom-functions` guide instead of duplicating the content